pkg/apis/resources/v1alpha2/register.go

```diff
@@ -217,6 +217,15 @@ func addWebService(c *restful.Container) error {
 		Reads(registriesmodel.AuthInfo{}).
 		Returns(http.StatusOK, ok, errors.Error{}))
 
+	webservice.Route(webservice.POST("registry/blob").
```

Comment:
what's this api used for?

Reply:
verb `POST` is usually used for creation or make a change, since this api is just retrieving information from server, I think `GET` is better here.

Reply:
When create deployment or other workloads, add container, will use this api for get image details. like: 
`zpl://screen?pid=5cbd3e2d21c1ba7ba1eb412a&sid=5d22ec8cd6d0da3769015c08`

at the view of theory, Maybe `GET` is better. But this api will receive a Secret, perhaps use `POST` and add the Secret in request body  is a good choice.

Reply:
where are these secrets from? aren't they created by users as imagePullSecrets, already in the cluster?

Reply:
you mean get secrets from cluster?

Reply:
yes, since imagePullSecrets already exists in cluster, there is no need for users to input again. they can just provide imagePullSecrets name or something, we do the rest for them.

Reply:
user only need to select a secret by name, and frontend will request image blob by secret which user selected, like this 
![image](https://user-images.githubusercontent.com/16031013/62912949-bf26c080-bdbc-11e9-939e-b75f1dabea92.png).
 if use `GET`, ks should requests secret from cluster by secret name and namespace, maybe this action is superfluous. 
how about frontend send secret data with request?

Reply:
no, it's not. With namespace and secret in request path, it's would be easy for iam to do authorization.

Reply:
yes